src/components/library-app-2/book-listing.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {Text, View, StyleSheet, ActivityIndicator} from 'react-native';
+import Book from './book';
+
+const {emptyArr} = global;
+
+const BookListing = ({books = emptyArr, isFetching = true}) => {
+  const {booksList} = styles;
+
+  return (
+    <View style={booksList}>
+      {isFetching ? (
+        <ActivityIndicator size="large" color="pink" />
+      ) : !!books.length ? (
```

Comment:
Try avoiding !! -> It causes double type casting. This purpose can also be solved using > 0 check.